src/api/private/notes/notes.controller.ts

```diff
@@ -174,6 +178,82 @@ export class NotesController {
     }
   }
 
+  @Post(':noteIdOrAlias/metadata/alias/:newAlias')
```

Comment:
I don't like the inconsistency in paths. At first, the relevant alias is at the end of the path, but for PUT and DELETE it is in the middle. For consistency, the `alias` that is operated on should always be at the end of the path.

Reply:
Unfortunately, this whole API is quite ugly.
- Normally, POST should operate on the collection directly, parameters (like the alias name) should be in the request body, not in the path.
- We are in the `/notes` namespace. At all other places, the `noteIdOrAlias` parameter in the path is used to find a *note*. If a note has 3 aliases, you can use them interchangeably and always operate in the same object. The PUT and DELETE operations here break this expectation, as they now use the parameter to identify an *alias*.

I think we should move the alias operations to a new `/aliases` route, to not create confusion here and have a cleaner API.